Incorrect imports in built sources

### Description
Sources generated with babel contain incorrect import paths.

### Only for bugs :bug:
**Version :** 1.0.0-rc.1
#### Expected behavior
...

#### Actual behavior
```
Module not found: Error: Can't resolve '../../../src/core/set' in './node_modules/immutadot/seq/core'
 @ ./node_modules/immutadot/seq/core/set.js 3:4-79
 @ ./node_modules/immutadot/seq/core/index.js
```

```
Module not found: Error: Can't resolve '../../../src/seq/ChainWrapper' in './node_modules/immutadot/seq/array'
 @ ./node_modules/immutadot/seq/array/push.js 3:4-67
 @ ./node_modules/immutadot/seq/array/index.js
```

```
Module not found: Error: Can't resolve 'core/add' in './node_modules/immutadot/seq/lang'
 @ ./node_modules/immutadot/seq/lang/add.js 3:4-66
 @ ./node_modules/immutadot/seq/lang/index.js
```

```
Module not found: Error: Can't resolve 'seq/core' in './node_modules/immutadot/util'
 @ ./node_modules/immutadot/util/protect.js 3:4-88
 @ ./node_modules/immutadot/util/index.js
```

--- a/.babelrc
+++ b/.babelrc
@@ -7,7 +7,7 @@
   ],
   "plugins": [
     ["module-resolver", {
-      "root": ["./src"]
+      "root": ["./src", "./generated"]
     }]
   ],
   "env": {
@@ -17,7 +17,7 @@
     "test": {
       "plugins": [
         ["module-resolver", {
-          "root": ["./src", "../../misc", "./generated"]
+          "root": ["./src", "./generated", "../../misc"]
         }]
       ]
     }

--- a/.babelrc
+++ b/.babelrc
@@ -5,14 +5,14 @@
     }],
     ["stage-2"],
   ],
-  "plugins": [
-    ["module-resolver", {
-      "root": ["./src", "./generated"]
-    }]
-  ],
   "env": {
     "production": {
-      "ignore": ".spec.js"
+      "ignore": ".spec.js",
+      "plugins": [
+        ["module-resolver", {
+          "root": ["./tmpsrc"]
+        }]
+      ],
     },
     "test": {
       "plugins": [

--- a/misc/generate-flow.js
+++ b/misc/generate-flow.js
@@ -85,7 +85,7 @@ ${nsItems.map(({ name }) => `  ${name},`).join('\n')}
             /* eslint-disable indent */
 `import { ChainWrapper } from 'seq/ChainWrapper'
 
-import { ${name} } from 'core/${name}'
+import { ${name} } from '${namespace}/${name}'
 
 ChainWrapper.prototype.${name} = function(path, ...args) {
   return this._call(${name}, path, args)

--- a/packages/immutadot-lodash/package.json
+++ b/packages/immutadot-lodash/package.json
@@ -17,8 +17,9 @@
   },
   "scripts": {
     "generate:flow": "node ../../misc/generate-flow.js",
-    "prebuild": "yarn generate:flow",
-    "build": "cross-env BABEL_ENV=production babel src generated -d .",
+    "prebuild": "yarn generate:flow && rm -rf tmpsrc && mkdir tmpsrc && cp -r src/* generated/* tmpsrc",
+    "build": "cross-env BABEL_ENV=production babel tmpsrc -d .",
+    "postbuild": "rm -rf tmpsrc",
     "prelint": "yarn generate:flow",
     "lint": "eslint src generated",
     "pretest": "lerna run --scope immutadot build && yarn generate:flow",

--- a/packages/immutadot/package.json
+++ b/packages/immutadot/package.json
@@ -30,8 +30,9 @@
   },
   "scripts": {
     "generate:flow": "node ../../misc/generate-flow.js",
-    "prebuild": "yarn generate:flow",
-    "build": "cross-env BABEL_ENV=production babel src generated -d .",
+    "prebuild": "yarn generate:flow && rm -rf tmpsrc && mkdir tmpsrc && cp -r src/* generated/* tmpsrc",
+    "build": "cross-env BABEL_ENV=production babel tmpsrc -d .",
+    "postbuild": "rm -rf tmpsrc",
     "prelint": "yarn generate:flow",
     "lint": "eslint src generated",
     "pretest": "yarn generate:flow",

